MergerLogic/ImageProcessing/TileMerger.cs

```diff
@@ -50,13 +49,17 @@ public TileMerger(ITileScaler tileScaler, ILogger<TileMerger> logger)
 
                             mergedImage.ColorSpace = ColorSpace.sRGB;
                             mergedImage.ColorType = mergedImage.HasAlpha ? ColorType.TrueColorAlpha : ColorType.TrueColor;
-                            ImageFormatter.ConvertToFormat(mergedImage, format);
-                            var mergedImageBytes = mergedImage.ToByteArray();
+                            image = new MagickImage(mergedImage);
                             this._logger.LogDebug($"[{MethodBase.GetCurrentMethod().Name}] 'imageMagic' merging finished");
-                            return mergedImageBytes;
                         }
                     }
+                    break;
             }
+
+            Tile tile = new Tile(targetCoords, image);
+            image.Dispose();
+            tile.ConvertToFormat(strategy.ApplyStrategy(tile.Format));
```

Comment:
i think that the tile.Format param is a bug, as this is not the "requested" format, you should use the Target format paramater that was introduced to the worker via task params